src/System.Runtime.Numerics/src/System.Runtime.Numerics.csproj

```diff
@@ -36,6 +36,7 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="project.json" />
+    <Compile Include="System\Numerics\BigIntegerCalculator.PowMod.cs" />
```

Comment:
Shouldn't this be in the ItemGroup above with the rest of the BigIntegerCalculator.* files?